js/bitmex.js

```diff
@@ -2645,6 +2649,18 @@ module.exports = class bitmex extends Exchange {
         return await this.privatePostPositionIsolate (this.extend (request, params));
     }
 
+    calculateRateLimiterCost (api, method, path, params, config = {}, context = {}) {
+        const isAuthenticated = this.checkRequiredCredentials (false);
+        if (config['cost'] !== 1) { // trading endpoints
```

Comment:
Can we protect this config access just in case someone adds a new endpoint in the future and forgets to add its weight/ config changes its format?

instead of `config['cost']` something like `this.safeInteger(confit, 'cost', 1)`. 

Do you agree?

Reply:
sure, although the config 'cost' will always be there, I believe it is added in the main `Exchange.js` class rather than in `bitmex.js`

Reply:
@DoctorSlimm yes it's added here, but for instance, if we changed this in the future to a new format that code would break 
![image](https://user-images.githubusercontent.com/43336371/181225842-1bbe010f-2a00-4cbf-83b0-ebdfb943674d.png)